client/src/Components/MemeBox/MemeBox.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect } from 'react';
+import { useSocket } from 'socketio-hooks';
+import { Modal, Backdrop, Fade } from '@material-ui/core';
+
+import MemeBoxStyles from './MemeBoxStyles';
+import { memeSchema } from './memeSchema';
+
+const MemeBox = () => {
+  const classes = MemeBoxStyles();
+  const [open, setOpen] = useState(false);
+  const [image, setImage] = useState(null);
+
+  useSocket('MEME', votes => {
+    const filteredVotes = filterOnlyNumbers(votes.map(user => user.vote));
+    if (filteredVotes?.length) {
```

Comment:
Myślę, że da się to prościej zapisać skoro mamy jednego ifa wewnątrz  drugiego ?